docs/configuration.html

```diff
@@ -70,9 +70,13 @@
 
 <h3><a id="consumerconfigs" href="#consumerconfigs">3.3 Consumer Configs</a></h3>
 
-We introduce both the old 0.8 consumer configs and the new consumer configs respectively below.
+We introduce both the new Java consumer configs and the older Scala consumer configs respectively below.
 
-<h4><a id="oldconsumerconfigs" href="#oldconsumerconfigs">3.3.1 Old Consumer Configs</a></h4>
+<h4><a id="newconsumerconfigs" href="#newconsumerconfigs">3.3.1 New Consumer Configs</a></h4>
+In 0.9.0.0 we introduced a replacement for simple and high-level consumers. Below is the configuration for the new consumer:
```

Comment:
Sounds good. Please check the variation I used in the updated PR and let me know if it needs further modification. Thanks.